.vale.ini

```diff
@@ -0,0 +1,7 @@
+StylesPath = "styles/camunda"
+
+[{docs,versioned_docs}/**/*.{md,mdx}]
+BasedOnStyles = docsInstance
+
+[{optimize,optimize_versioned_docs}/**/*.{md,mdx}]
+BasedOnStyles = optimizeInstance
```

Comment:
We can easily configure the ability to share rules across the docs instances, but for this prototype I only included unique rules for each instance.

Reply:
I love the flexibility here!